### DISCUSSION TITLE
Vulnerabilities no impact is for  OpenSSL 3.0.10 FIPS provider or OpenSSL 3.0.9 FIPS provider ?

### DISCUSSION THREAD

**Discussion by absrivas-in** [openssl/openssl#21655] [Q&A] [answered]
OpenSSL 3.0.10 fixes vulnerabilities mentioned in https://www.openssl.org/news/fips-cve.html
and states they don't impact FIPS.
So the question is the version of FIPS module to be used with OpenSSL 3.0.10 Is it OpenSSL 3.0.10 FIPS provider or OpenSSL 3.0.9 FIPS provider?

**Comment by mattcaswell** [comment, marked as answer]
From the Download page:

> Please follow the Security Policy instructions to download, build and install a validated OpenSSL FIPS provider. Other OpenSSL Releases MAY use the validated FIPS provider, but MUST NOT build and use their own FIPS provider. For example you can build OpenSSL 3.1 and use the OpenSSL 3.0.8 FIPS provider with it.

The *only* validated versions listed on the download page are 3.0.0 and 3.0.8. So use 3.0.10 for the libssl/libcrypto etc, but use the FIPS provider from 3.0.8

There is extensive discussion on how to do this in the README-FIPS.md file:

https://github.com/openssl/openssl/blob/openssl-3.0/README-FIPS.md
